### ISSUE TITLE
Not able to download the Pretrained ESG-BERT

### ISSUE THREAD

**Issue by UdiBhaskar** [mukut03/ESG-BERT#8]
https://drive.google.com/drive/folders/1yfNpMvByz3fJMsOqir3SerS6PwsRS2rt?usp=sharing   This link is giving error. Can you please update the link?

**Comment by pascalporedda** [comment]
@UdiBhaskar 

You have to look here: https://drive.google.com/drive/folders/1Qz4HP3xkjLfJ6DGCFNeJ7GmcPq65_HVe

You'll find that link a little bit lower in the README

**Comment by UdiBhaskar** [comment]
That is fine tuned for classification. but needed the pretrained(MLM) ESG Bert.

**Comment by satish-silveri** [comment]
Yes, can I also get the link for pre-trained(MLM) ESG BERT? The link (https://drive.google.com/drive/folders/1yfNpMvByz3fJMsOqir3SerS6PwsRS2rt?usp=sharing) mentioned in the readme file is not working.

Appreciate the help :)

**Comment by UdiBhaskar** [comment]
This is giving "404 Not Found" error.

**Comment by nadahlberg** [comment]
Bumping this -- does anyone have the weights for the MLM ESGBERT (not the classifier)
